flow/designs/asap7/mock-array/config.mk

```diff
@@ -23,11 +23,17 @@ export DIE_AREA  = $(shell \
   cd $(dir $(DESIGN_CONFIG)) && \
   python3 -c "import config; print(f'{0} {0} {config.die_width} {config.die_height}')")
 
+export MACRO_PLACE_HALO = 0 2.16
+export RTLMP_BOUNDARY_WT = 0
+export RTLMP_FLOW ?= True
+
 export BLOCKS                ?= Element
 
 ifneq ($(BLOCKS),)
   export GDS_ALLOW_EMPTY       = Element
-  export MACRO_PLACEMENT_TCL   = ./designs/asap7/mock-array/macro-placement.tcl
+  ifeq ($(RTLMP_FLOW), False)
```

Comment:
An niggling little thing about ORFS variables is that there are no types... RTLMP_FLOW actually checks for empty and non-empty....

Also, I would like to keep manual placement under CI.

See here on how to do that:

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/asap7/riscv32i-mock-sram/config.mk

1. create designs/asap7/mock-array-manual/config.mk
2. set RTLMP_FLOW to empty
3. include designs/asap7/mock-array/config.mk
4. add the baseline files(make metadata/update_ok/rules, etc.)

Reply:
Fewer surprises: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2208